.ci/build-win.sh

```diff
@@ -14,11 +14,9 @@ BUILD=build-win${ARCH}
 INSTALL=install-win${ARCH}
 DIST=harfbuzz-win${ARCH}
 
-rm -rf ${BUILD}
-
 meson setup \
 	--buildtype=release \
-	--prefix=${PWD}/${INSTALL} \
+	--prefix=${PWD}/${BUILD}/${INSTALL} \
```

Comment:
We might want to quote `${PWD}`, as some people have the habit of using paths with spaces.